README.md

```diff
@@ -36,17 +36,18 @@ in your browser. To force retrieval of the default conguration, use
 
 ![rake-eslint-rails-run][]
 
+This will analyze all of the javascript files in the project:
```

Comment:
This would require a major version bump, since it's backward incompatible. I would much prefer adding a new task, perhaps `eslint:run_all`, to run eslint on all files and leaving the default behavior as it is.